internal/cli/deployment_list.go

```diff
@@ -211,27 +215,33 @@ func (c *DeploymentListCommand) Run(args []string) int {
 
 			if user, ok := b.Labels["common/user"]; ok {
 				details = append(details, "user:"+user)
-			} else if user, ok := b.Preload.Build.Labels["common/user"]; ok {
-				details = append(details, "build-user:"+user)
-			}
+			} else if deployBundle.Build != nil {
+				build := deployBundle.Build
+				// preload labels have been set, safe to use them
 
-			if bp, ok := b.Preload.Build.Labels["common/languages"]; ok {
-				details = append(details, niceLanguages(bp))
-			}
+				if user, ok := build.Labels["common/user"]; ok {
+					details = append(details, "build-user:"+user)
+				}
+				if bp, ok := build.Labels["common/languages"]; ok {
+					details = append(details, niceLanguages(bp))
+				}
 
-			if img, ok := b.Preload.Build.Labels["common/image-id"]; ok {
-				img = shortImg(img)
+				if img, ok := build.Labels["common/image-id"]; ok {
+					img = shortImg(img)
 
-				details = append(details, "image:"+img)
+					details = append(details, "image:"+img)
+				}
 			}
 
-			artDetails := fmt.Sprintf("artifact:%s", c.flagId.FormatId(b.Preload.Artifact.Sequence, b.Preload.Artifact.Id))
-			if len(details) == 0 {
-				details = append(details, artDetails)
-			} else if c.flagVerbose {
-				details = append(details,
-					artDetails,
-					fmt.Sprintf("build:%s", c.flagId.FormatId(b.Preload.Build.Sequence, b.Preload.Build.Id)))
+			if deployBundle.Artifact != nil {
+				artDetails := fmt.Sprintf("artifact:%s", c.flagId.FormatId(deployBundle.Artifact.Sequence, deployBundle.Artifact.Id))
+				if len(details) == 0 {
+					details = append(details, artDetails)
+				} else if c.flagVerbose && deployBundle.Build != nil {
+					details = append(details,
+						artDetails,
+						fmt.Sprintf("build:%s", c.flagId.FormatId(deployBundle.Build.Sequence, deployBundle.Build.Id)))
+				}
 			}
 
 			if c.flagVerbose {
```

Comment:
I can't reach this comment on the PR, but there's a panic on line 246. It should use the deployment bundle instead of `b.Preload.Artifact`. I think the whole `-verbose` output needs this fix up.

Reply:
Oh thank you! I thought I found all the old uses of `Preload`, but I missed those _and_ neglected to test `verbose`. Fixing now.

Reply:
Hm, there's more. Still getting a panic on 260, and once that's resolved there will be another one on 270.

Reply:
Yup, just pushed fixes for the last of them.

I also updated the integration test to do a few list calls, in leiu of actual tests for the CLI commands, which should help catch future panics.